web/Areas/Directory/Controllers/DirectoryController.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Viper.Models.AAUD;
+using Viper.Models.RAPS;
+using Viper.Areas.RAPS.Services;
+using Web.Authorization;
+using Microsoft.IdentityModel.Tokens;
+using Viper.Classes;
+using Polly;
+using Viper.Classes.SQLContext;
+using Viper.Areas.RAPS.Models;
+using Viper.Areas.Directory.Models;
+using System;
+
+namespace Viper.Areas.Directory.Controllers
+{
+    //TODO: Create the Directory Delegate Users role and add anyone with access to delegate roles
+    [Area("Directory")]
+    [Authorize(Roles = "VMDO SVM-IT,RAPS Delegate Users", Policy = "2faAuthentication")]
```

Comment:
[Permission(Allow = "SVMSecure")] is probably a sufficient permission check. As it is now, this restricts the page to a much smaller set of users.